ui/src/message_popup/author/edit_question_page.jsx

```diff
@@ -2,7 +2,9 @@
 import React from 'react';
 import _ from 'lodash';
 
+import * as Api from '../../helpers/api.js';
 import {allQuestions} from '../questions.js';
+import {allArchivedQuestions} from './archived_questions.js';
 import {withStudents, withIndicator} from '../transformations.jsx';
 
 import QuestionPage from './question_page.jsx';
```

Comment:
Please update the component name and filename so they match, otherwise grepping and searching gets tricky :)